etc/win-ci/crystal.iss

```diff
@@ -0,0 +1,335 @@
+#define MyAppName "Crystal"
+#define VersionFile FileOpen("portable\src\VERSION")
+#define MyAppVersion FileRead(VersionFile)
+#define MyAppVersionNum StringChange(MyAppVersion, "-dev", "")
+#expr FileClose(VersionFile)
+#define MyAppPublisher "Manas Technology Solutions"
+#define MyAppURL "https://crystal-lang.org/"
+#define MyAppExeName "crystal.exe"
+#define MyAppAssocName MyAppName + " Source File"
+#define MyAppAssocExt ".cr"
+#define MyAppAssocKey StringChange(MyAppAssocName, " ", "") + MyAppAssocExt
+
+[Setup]
+AppId={{7C307DDF-447E-46C5-BB3B-47A6F652D7C8}

```

Comment:
I suppose the unbalanced curly brace is on purpose?

Reply:
[This is indeed how they escape the `{` character](https://jrsoftware.org/ishelp/topic_consts.htm):

> A "{" character is treated as the start of the constant. If you want to use that actual character in a place where constants are supported, you must use two consecutive "{" characters. (You do not need to double "}" characters.)